wire-ios-sync-engine/Source/Synchronization/Strategies/UserClientRequestStrategy.swift

```diff
@@ -259,6 +259,7 @@ public final class UserClientRequestStrategy: ZMObjectSyncStrategy, ZMObjectStra
 
         if keys.contains(ZMUserClientNeedsToUpdateSignalingKeysKey) {
             do {
+                // what are these: signaling keys? what is it for?
```

Comment:
@typfel I am trying to remove some files (ZMAPSMessageDecoder) you mention a while ago, is APSSignalingKeysStore still useful?